components/CancellationList.vue

```diff
@@ -142,7 +144,16 @@ export default {
         sortable: false,
       },
     ],
+    //SnackBar Notification Data
+    snackBarData: {
+      open: false,
```

Comment:
since this is moving to `data`, scope it to snackbar, like `showSnackBar` and `snackBarText`.